pkg/kritis/review/review_test.go

```diff
@@ -127,63 +145,103 @@ func TestReviewGAP(t *testing.T) {
 		image        string
 		policies     []v1beta1.GenericAttestationPolicy
 		attestations []metadata.PGPAttestation
-		isAttested   bool
+		isAdmitted   bool
 		shouldErr    bool
 	}{
 		{
 			name:         "valid image with attestation",
 			image:        img,
 			policies:     oneGAP,
-			attestations: validAtts,
-			isAttested:   true,
+			attestations: oneValidAtt,
+			isAdmitted:   true,
 			shouldErr:    false,
 		},
 		{
 			name:         "image without attestation",
 			image:        img,
 			policies:     oneGAP,
 			attestations: []metadata.PGPAttestation{},
-			isAttested:   false,
+			isAdmitted:   false,
 			shouldErr:    true,
 		},
+		{
+			name:         "gap without attestor should error",
+			image:        img,
+			policies:     gapWithoutAA,
+			attestations: []metadata.PGPAttestation{},
+			isAdmitted:   false,
+			shouldErr:    true,
+		},
+		{
+			name:         "gap without attestor should error on allowlisted image",
+			image:        "allowed_image_name",
+			policies:     gapWithoutAA,
+			attestations: []metadata.PGPAttestation{},
+			isAdmitted:   false,
+			shouldErr:    true,
+		},
+		{
+			name:         "allowlisted image",
+			image:        "allowed_image_name",
+			policies:     oneGAP,
+			attestations: []metadata.PGPAttestation{},
+			isAdmitted:   false,
+			shouldErr:    false,
+		},
+		{
+			name:         "image allowlisted in 1 policy",
+			image:        "allowed_image_name",
+			policies:     twoGAPs,
+			attestations: []metadata.PGPAttestation{},
+			isAdmitted:   false,
```

Comment:
ditto here, it's unclear why it wouldn't be admitted

Reply:
ditto as above.